test/oauth2Auth.spec.ts

```diff
@@ -22,7 +22,8 @@ import { EcmAuthMock, OAuthMock } from '../test/mockObjects';
 import { PathMatcher } from '../src/utils/path-matcher';
 import chai, { expect } from 'chai';
 
-// const jsdom = require('mocha-jsdom');
```

Comment:
needs to be removed from package.json?

Reply:
I think you've already removed it here: https://github.com/Alfresco/alfresco-js-api/pull/1678/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519